File: TS/problems.ts
function add(a: number, b: number): number { return a + b; }
function subtract(a: number, b: number): number { return a - b; }
function multiply(a: number, b: number): number { return a * b; }
function divide(a: number, b: number): number | null { return b !== 0 ? a / b : null; } // Add error handling for division by zero

const x = 10;
const y = 5;

console.log("Add:", add(x, y));
console.log("Subtract:", subtract(x, y));
console.log("Multiply:", multiply(x, y));
console.log("Divide:", divide(x, y) ?? "Error (division by zero)");

File: DART/problems.dart
int add(int a, int b) => a + b;
int subtract(int a, int b) => a - b;
int multiply(int a, int b) => a * b;
double? divide(int a, int b) => b != 0 ? a / b : null; // Add error handling for division by zero

void main() {
  int x = 10, y = 5;
  print("Add: ${add(x, y)}");
  print("Subtract: ${subtract(x, y)}");
  print("Multiply: ${multiply(x, y)}");
  print("Divide: ${divide(x, y)}");
}
